server/src/test/resources/config/sidecar_validation_configuration.yaml

```diff
@@ -25,11 +25,16 @@ sidecar:
     execute_interval: 50ms
 
 cassandra_input_validation:
+  validator:
+  - class_name: org.apache.cassandra.sidecar.utils.FastCassandraInputValidator
+    parameters:
+      valid_terminations: ".abc,.def"
+      valid_restricted_terminations: ".xml"
   forbidden_keyspaces:
     - a
     - b
     - c
   allowed_chars_for_directory: "[a-z]+"
   allowed_chars_for_quoted_name: "[A-Z]+"
-  allowed_chars_for_component_name: "(.db|.cql|.json|.crc32|TOC.txt)"
-  allowed_chars_for_restricted_component_name: "(.db|TOC.txt)"
+  allowed_chars_for_component_name: "(\\.db|\\.cql|\\.json|\\.crc32|TOC\\.txt)"
```

Comment:
Nit: Remove regex validator parameters since they are not relevant.

Reply:
We still use these values to validate in unit tests. See https://github.com/frankgh/cassandra-sidecar/blob/CASSSIDECAR-361/server/src/test/java/org/apache/cassandra/sidecar/config/SidecarConfigurationTest.java#L109